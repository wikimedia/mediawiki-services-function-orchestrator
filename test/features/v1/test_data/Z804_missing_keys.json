{
	"Z1K1": "Z7",
	"Z7K1": "Z804",
	"Z804K1": [
		{
			"Z1K1": "Z9",
			"Z9K1": "Z39"
		},
		{
			"Z1K1": {
				"Z1K1": "Z9",
				"Z9K1": "Z39"
			},
			"Z39K1": {
				"Z1K1": "Z6",
				"Z6K1": "Z2K1"
			}
		},
		{
			"Z1K1": {
				"Z1K1": "Z9",
				"Z9K1": "Z39"
			},
			"Z39K1": {
				"Z1K1": "Z6",
				"Z6K1": "Z8K2"
			}
		},
		{
			"Z1K1": {
				"Z1K1": "Z9",
				"Z9K1": "Z39"
			},
			"Z39K1": {
				"Z1K1": "Z6",
				"Z6K1": "Z8K6"
			}
		}
	],
	"Z804K2": {
		"Z1K1": "Z8",
		"Z8K1": [
			"Z17",
			{
				"Z1K1": "Z17",
				"Z17K1": {
					"Z1K1": "Z7",
					"Z7K1": "Z881",
					"Z881K1": "Z1"
				},
				"Z17K2": "Z813K1",
				"Z17K3": {
					"Z1K1": "Z12",
					"Z12K1": [
						"Z11",
						{
							"Z1K1": "Z11",
							"Z11K1": "Z1002",
							"Z11K2": "list"
						}
					]
				}
			}
		],
		"Z8K2": "Z40",
		"Z8K3": [
			"Z9",
			"Z8130",
			"Z8131"
		],
		"Z8K4": [
			"Z9",
			"Z913"
		],
		"Z8K5": "Z813"
	}
}
