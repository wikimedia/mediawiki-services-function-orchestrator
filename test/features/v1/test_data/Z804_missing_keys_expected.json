{
	"Z1K1": "Z5",
	"Z5K1": "Z507",
	"Z5K2": {
		"Z1K1": {
			"Z1K1": "Z7",
			"Z7K1": "Z885",
			"Z885K1": "Z507"
		},
		"Z507K1": "Object did not contain key(s): Z2K1,Z8K6"
	}
}
